tests/unit/Core/Framework/App/Lifecycle/Parameters/AppInstallParametersTest.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Unit\Core\Framework\App\Lifecycle\Parameters;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\App\Lifecycle\Parameters\AppInstallParameters;
+use Shopware\Core\Framework\Log\Package;
+
+/**
+ * @internal
+ */
+#[Package('framework')]
+#[CoversClass(AppInstallParameters::class)]
+class AppInstallParametersTest extends TestCase
```

Comment:
do we really need such test? :see_no_evil:

Reply:
for me it would also be fine to add the coverageIgnore annotation to those classes. I do not really see any value in testing DTOs

Reply:
removed